app/views/viewing_party/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Create a Viewing Party</h1>
+
+<%= form_tag viewing_party_index_path, method: :post, class: "new-viewing-party-form" do %>
+  <%= tag.h2 @movie.title %>
+  <%= hidden_field_tag :movie_title, @movie.title %>
+
+  <%= label_tag :party_duration %>
+  <%= text_field_tag :party_duration, @movie.runtime %><br/>
+
+  <label for="date">Choose a date for your party:</label>
```

Comment:
I liked this - didn't know that this was a possibility to add straight HTML to our html.erb pages